vendor.conf

```diff
@@ -37,3 +37,4 @@ github.com/Microsoft/go-winio v0.4.1
 github.com/boltdb/bolt e9cf4fae01b5a8ff89d0ec6b32f0d9c9f79aefdd
 github.com/Microsoft/hcsshim v0.5.15
 github.com/Azure/go-ansiterm fa152c58bc15761d0200cb75fe958b89a9d4888e
+github.com/AkihiroSuda/filegrain c017b52c51386e287a61e48122e9ca2921563c48
```

Comment:
This vendoring is solely for my generic OCI library
https://github.com/AkihiroSuda/filegrain/tree/c017b52c51386e287a61e48122e9ca2921563c48/image

I can unvendor this pkg if I should do so (`gitHub.com/containerd/containerd/ociimage`?)